activiti-core/activiti-engine/src/main/java/org/activiti/engine/impl/bpmn/helper/ErrorPropagation.java

```diff
@@ -276,65 +276,98 @@ protected static Map<String, List<Event>> findCatchingEventsForProcess(
         String processDefinitionId,
         String errorRef
     ) {
-        Map<String, List<Event>> eventMap = new HashMap<String, List<Event>>();
+        Map<String, List<Event>> eventMap = new HashMap<>();
         Process process = ProcessDefinitionUtil.getProcess(processDefinitionId);
         BpmnModel bpmnModel = ProcessDefinitionUtil.getBpmnModel(processDefinitionId);
 
         String compareErrorCode = retrieveErrorCode(bpmnModel, errorRef);
 
+        // Process event sub-processes
```

Comment:
This comment means that the following block can be extracted in a method with similar name. In this case you can extract `findCatchingEventSubprocesses`